### PR TITLE
api-config-cleanup

### PR DESCRIPTION
**Pull request by maaikelimper**
Hi Tom,

I would like to cleanup the pygeoapi-config.yml, could you please help review via this PR and provide suggestions for improvement ?

### CODE REVIEW

File: docker/pygeoapi-config.yml [hunk 1]
```diff
@@ -26,70 +26,31 @@ logging:
...
-        phone: +xx-xxx-xxx-xxxx
-        fax: +xx-xxx-xxx-xxxx
-        email: you@example.org
-        url: https://example.org
-        hours: Mo-Fr 08:00-17:00
-        instructions: During hours of service. Off on weekends.
-        role: pointOfContact
+        name: none
+        email: none
+    license:
```
Comment (tomkralidis) [inline docker/pygeoapi-config.yml:49]: This should point to information for licensing; the WMO Unified Data Policy would be the safest value to encompass datasets published through wis2box (instance wide).  Note that this value is displayed on the API landing page.

File: docker/pygeoapi-config.yml [hunk 2]
```diff
@@ -26,70 +26,31 @@ logging:
...
+            en: wis2box-api
         description:
             en: |-
-                WIS 2.0 in a box provides a platform with the capabilities
+                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                 for centres to integrate their data holdings and publish them
                 to the WMO Information System with a plug and play capability
-                supporting data publishing, discovery and access
+                supporting data publishing, discovery and access.
+                The wis2box-api provides processes to manage datasets and tools
```
Comment (tomkralidis) [inline docker/pygeoapi-config.yml:36]: WIS 2.0 in a box (wis2box) provides a platform with the capabilities
for centres to integrate their data holdings and publish them
to the WMO Information System with a plug and play capability
supporting data publishing, discovery and access.
wis2box-api provides an API providing data and processes for interactive visualization, data transformation and publication.

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: docker/pygeoapi-config.yml (hunk 1 above)
Comment (maaikelimper) [reply docker/pygeoapi-config.yml:49]: @tomkralidis can you please provide a proposal that would be a valid placeholder for anyone running a wis2box ?

**Comment by maaikelimper** [comment]
screenshot from local test with latest changes:
image

File: docker/pygeoapi-config.yml (hunk 1 above)
Comment (tomkralidis) [reply docker/pygeoapi-config.yml:49]: Below, for consideration/feedback:

```yaml
    identification:
        title:
            en: WIS2 in a box API
        description:
            en: |-
                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                for centres to integrate their data holdings and publish them
                to the WMO Information System with a plug and play capability
                supporting data publishing, discovery and access.
                wis2box-api provides an API providing data and processes for interactive
                visualization, data transformation and publication.
        keywords:
            en:
                - wis2box
                - WIS2 Node
        url: https://docs.wis2box.wis.wmo.int
    provider:
        name: WIS 2.0 in a box
        url: https://docs.wis2box.wis.wmo.int
    contact:
        name: World Meteorological Organization
        address: 7bis, avenue de la Paix
        city: Geneva
        postalcode: 1211
        country: Switzerland
        email: null
        url: https://wmo.int/contact-us
        role: pointOfContact
    license:
        name: WMO Unified Data Policy
        url: https://library.wmo.int/records/item/58009-wmo-unified-data-policy
```

Comment (maaikelimper) [reply docker/pygeoapi-config.yml:49]: @tomkralidis we can not hardcode WMO as contact into the base-api-config, I prefer to leave the contact empty, I have implemented the license as discussed

Comment (tomkralidis) [reply docker/pygeoapi-config.yml:49]: Updated proposal (difference being Apache 2.0 as license):

```yaml
    identification:
        title:
            en: WIS2 in a box API
        description:
            en: |-
                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                for centres to integrate their data holdings and publish them
                to the WMO Information System with a plug and play capability
                supporting data publishing, discovery and access.
                wis2box-api provides an API providing data and processes for interactive
                visualization, data transformation and publication.
        keywords:
            en:
                - wis2box
                - WIS2 Node
        url: https://docs.wis2box.wis.wmo.int
    provider:
        name: WIS 2.0 in a box
        url: https://docs.wis2box.wis.wmo.int
    contact:
        name: World Meteorological Organization
        address: 7bis, avenue de la Paix
        city: Geneva
        postalcode: 1211
        country: Switzerland
        email: null
        url: https://wmo.int/contact-us
        role: pointOfContact
    license:
        name: Apache License 2.0
        url: https://www.apache.org/licenses/LICENSE-2.0
```

**Comment by maaikelimper** [comment]
I set the e-mail to "not-defined" as setting it to None leads to errors when the collections are added

**Pull request merged**